models/user.js

```diff
@@ -0,0 +1,25 @@
+module.exports = function(sequelize, DataTypes) {
+    const User = sequelize.define("User", {
+      user_name: {
+        type: DataTypes.STRING,
+        allowNull: false,
+        validate: {
+          len: [1]
+        }
+      },
+      email: {
+        type: DataTypes.STRING,
+        allowNull: false,
+
+      },
+      user_password: {
+        type: DataTypes.STRING,
```

Comment:
if you are ensuring that it is max 14 characters we shouldn't leave the type at STRING instead we should use STRING(14)